dashboard/app/views/lti/v1/account_linking/landing.haml

```diff
@@ -1,8 +1,8 @@
 - content_for(:head) do
   :ruby
     script_data = {}
-    script_data[:lti_provider] = params[:lti_provider]
-    script_data[:lti_provider_name] = Policies::Lti::LMS_PLATFORMS[params[:lti_provider].to_sym][:name]
+    script_data[:lti_provider] = lti_provider
```

Comment:
Nice, I had to make these same changes in my PR to get these params to pass properly. Good to see it wasn't just my machine

Reply:
Yeah, took me a minute and had to learn what this line was doing (setting the locals values) - https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/lti_v1_controller.rb#L202

Reply:
My assumption is that's the correct way to pass variables to the view in a redirect, where setting an instance variable probably isn't accessible in the view due to the redirect? 🤷‍♂️